website/docs/basics/helpers-javascript.md

```diff
@@ -0,0 +1,311 @@
+  }, 250);
+```
+
+## devices
+
+Check if your browser's navigator is specific device.
+
+**Usage:**
+
+```js
+  import { device } from '@eightshift/frontend-libs/scripts/helpers';
+
+  device.iPhone();
+```
+
+## dynamicImport
+
+To get (require) all the files using the `require.context` method. It will find all files recursively in the folder using a regex. The following example will require all assets/index.js files inside the custom folder, so there is no need to manually add the files to the build.
+
+ * @param {object} paths All require.context patch to iterate.
+
+**Usage:**
+
+```js
+  import { dynamicImport } from '@eightshift/frontend-libs/scripts/helpers';
+
+  dynamicImport(require.context('./../../custom', true, /assets\/index.js$/));
+```
+
+## escape-string
+
+Takes provided string and removes special characters. Characters that will be removed: `([;&,.+*~':"!^#$%@[\]()=>|]`.
+
+* @param string $key Key to check.
+
+**Usage:**
+
+```js
+  import { escapeString } from '@eightshift/frontend-libs/scripts/helpers';
+
+  escapeString.escapeString('Special string');
+```
+
+## responsiveSelectors
+
+Create responsive selectors used for responsive attributes.
+
+* @param array   items        Array of breakpoints.
+* @param string  selector     Selector for this breakpoint.
+* @param string  parent       Parent block selector.
+* @param boolean use_modifier If false, you can use this selector for visibility.
+
+**Usage:**
+
+```js
+  import { responsiveSelectors } from '@eightshift/frontend-libs/scripts/helpers';
+
+  responsiveSelectors($attributes['width'], 'width', $blockClass);
+```
+
+**Output:**
+
+```js
+  .block-column__width-large--4
+```
+
+## selectorB
+
+Return's BEM selector for HTML class and checks if block is set.
+
+* @param string block BEM Block selector. *(required)*
+* @param string element BEM Element selector.
+* @param string modifier BEM Modifier selector.
+
+**Usage:**
+
+```js
+  import { selectorB } from '@eightshift/frontend-libs/scripts/helpers';
+
+  selectorB(blockClass);
+  selectorB(blockClass, selectorClass);
+  selectorB(blockClass, selectorClass, modifierClass);
+```
+
+**Equivalent:**
+
+```js
+  blockClass ? `${blockClass}` : '';
+  blockClass ? `${blockClass}__${selectorClass}` : '';
+  blockClass ? `${blockClass}__${selectorClass}--${modifierClass}` : '';
+```
+
+
+## selectorE
+
+Return's BEM selector for HTML class and checks if the element is set.
+
+* @param string block BEM Block selector. *(required)*
+* @param string element BEM Element selector. *(required)*
+* @param string modifier BEM Modifier selector.
+
+**Usage:**
+
+```js
+  import { selectorE } from '@eightshift/frontend-libs/scripts/helpers';
+
+  selectorE(blockClass, selectorClass);
+  selectorE(blockClass, selectorClass, modifierClass);
+```
+
+**Equivalent:**
+
+```js
+  selectorClass ? `${blockClass}__${selectorClass}` : '';
+  selectorClass ? `${blockClass}__${selectorClass}--${modifierClass}` : '';
+```
+
+## selectorM
```

Comment:
Check the above. 

I know naming things is hard, but the names should be descriptive, especially in a 3rd party library.